### ISSUE TITLE
Multiple tabs open for online users causes network requests to fail

### ISSUE THREAD

**Issue by mrjones-plip** [medic/cht-core#7716]
**Describe the bug**
I'm the kind of person that looooves to have multiple tabs open for any project I'm working. For some time now when working on the CHT with online users I would notice some time the app would just stop responding. I couldn't quite figure out why until today.  When I open 3 tabs, the third one becomes unresponsive for specific AJAX requests.

**To Reproduce**
1. launch the cht docker helper 
2.  upgrade to 3.16 Beta 2
3. create [a hierarchy and some contacts](https://docs.communityhealthtoolkit.org/apps/tutorials/contact-and-users-1/#1-create-new-health-facility)
4. log in as the `medic` user, open two cht core tabs and one admin tab
5. on the Admin tab, click on "Users" (`/admin/#/users`)
6. Note that no users load, a spinner is shown

**Expected behavior**
A list of users load

**Logs**
NA

**Screenshots**
Here's a video of the basic block. Note that as soon as the interstitial tab is closed, the admin tab becomes unblocked an immediately proceeds to finish downloading users:

https://user-images.githubusercontent.com/8253488/183219097-a37f0120-66d9-43eb-aeec-7a714a8d18cb.mp4

Oddly, if you navigate back and forth the problem persists, but if you reload you get a brief error "Error Fetching Users" but then the users load:

https://user-images.githubusercontent.com/8253488/183219227-7f8f717c-6c5a-49af-b3b8-6a0cf4f1f71f.mp4

**Environment**
- Instance: local dev instance via [CHT Docker Helper](https://docs.communityhealthtoolkit.org/apps/guides/hosting/app-developer/#cht-docker-helper) 
- Browser: Firefox 103.0 (64-bit)
- Client platform: Ubuntu 20.04
- App: webapp
- Version: 3.16 Beta 2

**Additional context**
* The same behavior can be seen in the user area too:

https://user-images.githubusercontent.com/8253488/183221896-c70b2dc9-9d06-464b-9e09-89a6cf0e71a5.mp4

* This problem could be reproduced in Chrome (103.0.5060.134 (Official Build) (64-bit) ), but not with the same ease, it took a bit of nudging:

https://user-images.githubusercontent.com/8253488/183219841-2468fb68-c232-45ea-ada7-772ccda17ee0.mp4

**Comment by dianabarsan** [comment]
Browsers have a maximum number of concurrent connections to a given host. Chrome has 6. 
https://stackoverflow.com/questions/985431/max-parallel-http-connections-in-a-browser

It's expected that, when you exhaust the slots, all extra requests will be queued.

**Comment by mrjones-plip** [comment]
Ah - that makes sense!  Thanks for the speedy response. 

Closing, but trust if we want to work on a "all connections slots exhausted, please wait until connections complete" type of feature we'll re-open (but seems out of scope, given our offline first focus).
